pages/1.11/security/ent/secrets/secrets-api/index.md

```diff
@@ -94,11 +97,42 @@ curl -H "Authorization: token=$(dcos config show core.dcos_acs_token)"
 Authentication tokens expire after five days by default. If your program needs to run longer than five days, you will need a service account. Please see [Provisioning custom services](/1.11/security/ent/service-auth/custom-service-auth/) for more information.
 
 
+## Text and binary secrets
+
+From DC/OS Enterprise version 1.11 the Secrets API supports text and binary secrets. A text secret is a text value encoded with UTF-8 encoding. The Secrets API transparently passes through this value to a secret store backend. A binary secret is automatically encoded with base64 codec before it is passed to a secrets store backend.
```

Comment:
What does "transparently" mean in this context?

Reply:
Changed to:

> The Secrets API passes through this value to a secret store backend without any modification.